### DISCUSSION TITLE
volfrac key

### DISCUSSION THREAD

**Discussion by mumayy** [neperfepx/neper#440] [Q&A] [answered]
I could not find an example of usage of "volfrac" command in Neper. Can someone help me about it?

**Comment by rquey** [comment, marked as answer]
Here is an example: 
```
neper -T -n 10 -domain "cube(0.5,0.5,0.5)"  -for tesr -group "id<=5?1:2" -statgroup volfrac
```

It defines groups and requests their `volfrac`.

Note that it works only for a tesr.

**Reply by mumayy** [reply]
Thank youu
